internal/scrape/jitsi.go

```diff
@@ -0,0 +1,84 @@
+package scrape
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+	"regexp"
+
+	"github.com/mxschmitt/playwright-go"
+	"github.com/syncfast/clockwise/internal/tui"
+)
+
+// getParticipants retrieves the total participant count from a specified Jitsi
+// URL. It runs in a loop and updates the passed in `Data` struct every
+// `refreshInterval` seconds.
+func GetParticipants(url string, refreshInterval int, data *tui.Data, pw *playwright.Playwright) error {
```

Comment:
We'll probably need to rename both this and the parallel function in `zoom.go` to differentiate them and specify platform. Something like `scrapeZoom` and `scrapeJitsi` would probably work.

Reply:
Done